docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,171 @@
+
+**Note**: For enforcing exactly once semantics a table named 'dt_meta' must exist in the database. The sample SQL to create the same is as follows
+```
+"CREATE TABLE IF NOT EXISTS dt_meta (dt_app_id VARCHAR(100) NOT NULL, dt_operator_id INT NOT NULL, dt_window BIGINT NOT NULL, UNIQUE(dt_app_id,dt_operator_id,dt_window))".
+```
+**Note**: Additionally this operator assumes that the underlying database/table in which records are to be added supports transactions. If the database/table does not support transactions then a tuple may be inserted in a table more than once in case of auto recovery from a failure (violation of exactly once semantics).
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***0.9.4***
+3. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcTransactionableOutputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcTransactionableOutputOperator.html)***
+
+## How to Use?
+ Concrete subclasses need to choose an appropriate class (such as String or an appropriate concrete java class). Also implement a couple of abstract methods (if not using AbstractJdbcPOJOOutputOperator): `setStatementParameters(PreparedStatement statement, T tuple)` to set the parameter of the insert/update statement (which is a PreparedStatement) with values from the tuple and `getUpdateCommand()` the sql statement to update a tuple in the database. Note that these methods are also defined in AbstractJdbcPOJOOutputOperator.
+
+Several properties are available to configure the behavior of this operator and they are summarized in the table below.
+### <a name="AbstractJdbcTransactionableOutputOperatorProps"></a>Properties of AbstractJdbcTransactionableOutputOperator
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *batchSize* | Maximum number of tuples to insert in a single call (see explanation above). | int | No | 1000 |
+
+#### <a name="JdbcTransactionalStore"></a>Properties of JDBC Store (BackendStore)
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *databaseDriver* |JDBC Driver class for connection to JDBC Store. This driver should be there in class path | String | Yes | N/A |
+| *databaseUrl* | Database url of the form jdbc:subprotocol:subname | String | Yes | N/A |
```

Comment:
url => URL

What is "subname" ? The actual URL provided in the example below does not match this pattern. Please locate a defining spec on the net, use the pattern from it, and provide a link to it.